client/src/main/java/com/battleships/connection/commands/server/commands/FleetSunk.java

```diff
@@ -0,0 +1,37 @@
+package com.battleships.connection.commands.server.commands;
+
+import com.battleships.Translator;
+import com.battleships.connection.Connection;
+import com.battleships.connection.commands.AbstractServerCommand;
+import javafx.application.Platform;
+import javafx.scene.control.Alert;
+import javafx.scene.control.ButtonType;
+
+import java.util.Optional;
+
+public class FleetSunk extends AbstractServerCommand {
+    public FleetSunk(Object value) {
+        super(value);
+    }
+
+    @Override
+    public void execute() {
+        Platform.runLater(() -> showWinnerAlertAndDisconnectPlayerOnPressedButtonOk());
+    }
+
+    private void showWinnerAlertAndDisconnectPlayerOnPressedButtonOk() {
+        Alert alert = new Alert(Alert.AlertType.INFORMATION);
+        alert.setHeaderText(null);
```

Comment:
Is this element setHeader necessary if it is null? Also i would refactor this whole method like this:
```
    private void showWinnerAlertAndDisconnectPlayerOnPressedButtonOk() {
        Alert alert = prepareAlert();
        Optional<ButtonType> result = alert.showAndWait();
        exitApplication(result);
    }
    
    private Alert prepareAlert() {
        Alert alert = new Alert(Alert.AlertType.INFORMATION);
        alert.titleProperty().bind(Translator.createStringBinding("game_won"));
        alert.contentTextProperty().bind(Translator.createStringBinding("game_won_info"));
        return alert;
    }
    
    private void exitApplication(Optional<ButtonType> result) {
        if (result.isPresent()) {
            Connection.INSTANCE.disconnect();
            Platform.exit();
        }
    }
```
Result has no meaning. It is not necessary to check if button which was passed is ok, or cancel.

Reply:
it is necesssary, because we don't want to show default header
thanks for the suggestion, but it doesn't work as we want